pkg/framework/plugins/removepodsviolatingnodetaints/node_taint_test.go

```diff
@@ -322,6 +336,41 @@ func TestDeletePodsViolatingNodeTaints(t *testing.T) {
 			expectedEvictedPodCount: 0, // p2 and p7 can't be evicted
 			nodeFit:                 true,
 		},
+		{
+			description:             "Pods tolerating included taints should not get evicted",
+			pods:                    []*v1.Pod{p1},
+			nodes:                   []*v1.Node{node1},
```

Comment:
s/node1/node7
This way the test can validate `testingTaint1=testing1` taint has no effect on the eviction.

Reply:
Yes, this is a nice double check. Changed it.